ide/dune

```diff
@@ -10,6 +10,8 @@
 
 (executable
  (name fake_ide)
+ (public_name fake_ide)
```

Comment:
What is this? I thought you had backtracked on this? (This seems confirmed by the way you have written the dune file BTW.

Reply:
I am still undecided on that one, binaries on path have certain advantages, and the main makefile expects fake_ide to be in path. I still got some time tho as this PR depends on other 3 so we can implement the solution we think best.

Reply:
If you are still undecided, then keep it uninstalled.